src/actions/createUserActions.js

```diff
@@ -1,78 +1,94 @@
+//     ok += "Ati introdus emailul gresit" + "\n";
+//   if (!/^([0]{1})\d{5,9}$/.test(body.phone))
+//     ok += "Ati introdus gresit numarul de telefon";
+//   if (body.email.length > 0 && checkMail(body, users) === 1) {
+//     ok += "Mailul folosit exista deja in baza de date";
+//   }
+//   if (ok === "") return 1;
+//   else return ok;
+// };
 
-        writeFile("db/users.json", JSON.stringify(users))
-          .then(() => {
-            let sentlink = body;
-            sentlink.link = {
-              GET: `/users/${sentlink.id}`,
-              PUT: `/users/${sentlink.id}`,
-              PATCH: `/users/${sentlink.id}`,
-              DELETE: `/users/${sentlink.id}`
-            };
-            res.status(200).send(sentlink);
-          })
-          .catch(error => {
-            response.status(500);
-            response.json({
-              serverErrorMessage:
-                "the error was logged and we’ll be checking it shortly"
-            });
-          });
-      }
-    })
-    .catch(error => {
-      response.status(500);
-      response.json({
-        serverErrorMessage:
-          "the error was logged and we’ll be checking it shortly"
-      });
-    });
-};
+// let checkMail = (newMail, allMails) => {
+//   for (let i = 0; i < allMails.length; i++) {
+//     if (newMail.email === allMails[i].email) return 1;
+//   }
+//   return 0;
+// };
+// if (process.env.NODE_ENV == "dev") {
+//   module.exports = {
+//     createUser,
+//     dataValid,
+//     checkMail
+//   };
+// } else {
+//   module.exports = {
+//     createUser
+//   };
+// }
 
-let dataValid = (body, users) => {
-  let ok = "";
-  if (!/^[a-zA-Z]+$/.test(body.firstName))
-    ok += "Ati introdus prenumele gresit" + "\n";
-  if (!/^[a-zA-Z]+$/.test(body.lastName))
-    ok += "Ati introdus numele gresit" + "\n";
-  if (!/^([a-z0-9A-Z])+\@([a-z0-9])+\.([a-z])+$/.test(body.email))
-    ok += "Ati introdus emailul gresit" + "\n";
-  if (!/^([0]{1})\d{5,9}$/.test(body.phone))
-    ok += "Ati introdus gresit numarul de telefon";
-  if (body.email.length > 0 && checkMail(body, users) === 1) {
-    ok += "Mailul folosit exista deja in baza de date";
-  }
-  if (ok === "") return 1;
-  else return ok;
-};
 
-let checkMail = (newMail, allMails) => {
-  for (let i = 0; i < allMails.length; i++) {
-    if (newMail.email === allMails[i].email) return 1;
-  }
-  return 0;
-};
-if (process.env.NODE_ENV == "dev") {
-  module.exports = {
-    createUser,
-    dataValid,
-    checkMail
-  };
-} else {
-  module.exports = {
-    createUser
-  };
-}
\ No newline at end of file
+const User = require('../models/userModel')
+const { sendErrorMessage, sendSuccessMessage } = require('../helpers/helpers')
+
+let createUser = (req, res) => {
+  let newUser = new User(req.body)
+  newUser.save((err, data) => {
+    if (err) console.log(err);
+    else res.status(200).send(data);
+  })
+}
+
+module.exports = {
+  createUser
+}
```

Comment:
Why are you leaving commented code in the project? If the purpose is to know what was written before, you can look in older commits.